js/components/App.js

```diff
@@ -24,6 +38,8 @@ export default Relay.createContainer(App, {
             node {
               id,
               name,
+              enabled,
+              ${ToggleWidgetEnableMutation.getFragment('widget')},
```

Comment:
Getting a warning when removing this line:

    Warning: RelayMutation: Expected prop `widget` supplied to `ToggleWidgetEnableMutation` to be data fetched by Relay. This is likely an error unless you are purposely passing in mock data that conforms to the shape of this mutation's fragment.